src/_base/docker/image/console/Dockerfile.twig

```diff
@@ -2,6 +2,11 @@ FROM {{ @('docker.image.console') }}
 # fix upstream signal
 STOPSIGNAL SIGTERM
 
+{% for service in [@('services.redis'), @('services.redis_session')] | map(v => v.enabled) | slice(0, 1) %}
+# copy the redis-cli from the first enabled redis service image
+COPY --from={{ service.image }} /usr/local/bin/redis-cli /usr/local/bin/redis-cli
+{% endfor %}
```

Comment:
I'm not sure on this, in that do we want to care what happens if a different image is used that doesn't have redis-cli in that location?

Reply:
In that instance I think overriding this template on the project would be okay.

Alternatively we can do a FROM build stage where it does `mv "$(which redis-cli)" /usr/local/bin/redis-cli`

Reply:
> I'm not sure on this, in that do we want to care what happens if a different image is used that doesn't have redis-cli in that location?

Also what if the other image has a different redis version in it (which is currently technically allowed)?